keychain_unix.cpp

```diff
@@ -76,37 +82,68 @@ static DesktopEnvironment detectDesktopEnvironment() {
     return DesktopEnv_Other;
 }
 
-static KeyringBackend detectKeyringBackend()
+static KeyringBackend detectKeyringBackend(void)
 {
+#if QT_VERSION >= 0x050000
+	const QString prefix = "qt5-";
+#else
+	const QString prefix = "qt4-";
```

Comment:
I don't think we need this prefix.
The settings should apply to both version equally. 

Also, there should not be so many apps using Qt4 at this point anywyay.